### DISCUSSION TITLE
Server list sortable groups

### DISCUSSION THREAD

**Discussion by faab007nl** [pterodactyl/panel#3266] [Feature Requests]
It would be nice if we could add server to groups and collapse the groups to make the panel more organized.

For example there is a page where u can make groups and the u can assign a server to a group. All servers with the same group go in the same dropdown/accordion.

**Comment by trenutoo** [comment]
There is an open request for this #498 and #1439

**Comment by mAsif12we** [comment]
**_Faisal_**
